docs/release.md

```diff
@@ -0,0 +1,92 @@
+# Releasing a new version of the Point of Sale Application
+
+All releases of this sample application are handled using CloudBuild triggers
+set up in the [`point-of-sale-ci`](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) GCP project. We use the custom written [releaser.py](/.github/releases/releaser.py) script for updating the configuration files with the correct
```

Comment:
maybe we should add a note for external contributors that this is only accessible to Googlers?

Reply:
Added note in b7af7f5ea4b86f196cd36442d3bc2ce4309601b8